### ISSUE TITLE
_addRowCallback is not called at all

### ISSUE THREAD

**Issue by Etoneja** [KasperOlesen/DataTable-AltEditor#98]
Hello!

https://github.com/KasperOlesen/DataTable-AltEditor/blob/d0f3e538c80e7a33d06e4204a774036a3dc239db/src/dataTables.altEditor.free.js#L686

Here you need to call checkFilesQueued();

Without this call, an ajax request will not be made.

**Comment by johnnyrusso** [comment]
Thank you. This took me WAY longer than it should have to figure out.

**Comment by luca-vercelli** [comment]
Thank you, I have introduced this regression last week. It should work now.

**Comment by Etoneja** [comment]
Thank you for your contribution!

Over the previous couple of days, I spent quite a lot of time with this library, customizing it for myself.  It is quite convenient to configure and has great potential.

Most likely, I will not dare to publish my local developments, since they are at the draft level.

I will describe a few moments/idead that I had to solve. Maybe someone wants to add them.
- special argument: if no options for select presented, take options from unique column values;
- take care about first empty placeholder (if not select2 used)
- css classes for textareas
- radiobuttons.
